lib/deploy.js

```diff
@@ -20,8 +24,7 @@ function sftpDeploy(opts) {
 
     tessel.logs.info('Bundling up code...');
     // Command V2 to extract a tarball it receives on stdin to the remote deploy dir
-    conn.exec('mkdir -p /tmp/remote-script/; rm -rf /tmp/remote-script/*; tar -x -C /tmp/remote-script', function(err, rstdin) {
-
+    conn.exec('mkdir -p ' + filepath + '/; rm -rf ' + filepath + '/*; tar -x -C ' + filepath, function(err, rstdin) {
```

Comment:
You should `rm -rf` the directory, then mkdir, or delete with `find`, because `*` will miss dotfiles, and requires the command to be executed in a shell.